dev/CommonStyles/InteractionTests/CommonStylesTests.cs

```diff
@@ -157,6 +157,12 @@ public void InkToolbarTest()
             {
                 var textBlock = FindElement.ByName("InkToolbar");
                 Verify.IsNotNull(textBlock, "Verify InkToolbar page doesn't crash");
+
+                var verticalSupport = FindElement.ByName("VerticalInkToolbar");
+                Verify.IsNotNull(verticalSupport, "Verify InkToolbar page doesn't crash for vertical support test");
+
+                DropDownButton flyoutButton = FindElement.ByName<DropDownButton>("InkToolbar");
```

Comment:
In truth, you haven't open flyout menu here.

Reply:
Why use DropDownButton here? I believe you could use var here, or use Inktoolbar sepcific control.